docs/metrics.md

```diff
@@ -265,6 +265,23 @@ Suppressed runs due to `concurrencyPolicy: Forbid` will not be counted.
 | `name`     | ⚠️ The name of the CronWorkflow |
 | `namespace` | The namespace of the CronWorkflow |
 
+#### `deprecated_feature`
+
+A counter which goes up when a feature which is [deprecated](deprecations.md) is used.
+🚨 This counter may go up much more than once for a single use of the feature.
+
+| attribute   | explanation                                 |
+|-------------|---------------------------------------------|
+| `feature`   | The name of the feature used                |
+| `namespace` | The namespace of the item using the feature |
+
+`feature` will be one of:
+
+- [`cronworkflow schedule`](deprecations.md#cronworkflow_schedule)
+- [`synchronization mutex`](deprecations.md#synchronization_mutex)
+- [`synchronization semaphore`](deprecations.md#synchronization_semaphore)
+- [`workflow podpriority`](deprecations.md#workflow_podpriority)
```

Comment:
```suggestion
- [`cronworkflow schedule`](deprecations.md#cronworkflow-schedule)
- [`synchronization mutex`](deprecations.md#synchronization-mutex)
- [`synchronization semaphore`](deprecations.md#synchronization-semaphore)
- [`workflow podpriority`](deprecations.md#workflow-podpriority)
```

- c.f. https://argo-workflows.readthedocs.io/en/release-3.6/deprecations/#cronworkflow-schedule
- hyphens are used, not underscores, same as https://github.com/argoproj/argo-workflows/pull/13739#discussion_r1815494815